### PR TITLE
Access module 

### PR DESCRIPTION
**Pull request by grafixeyehero**

### CODE REVIEW

**Comment by khimaros** [comment]
@grafixeyehero -- are you still planning to merge this?

**Comment by thornbill** [comment]
@jellyfin-bot rebase

File: .eslintrc.js [hunk 1]
```diff
@@ -81,6 +81,7 @@ module.exports = {
                 'jQuery': 'readonly',
                 // Jellyfin globals
                 'ApiClient': 'writable',
+                'events': 'writable',
```
Comment (thornbill) [inline .eslintrc.js:84]: I think using `Events` might be more consistent with the other module names that are exported?

File: src/components/directorybrowser/directorybrowser.js [hunk 2]
```diff
@@ -243,70 +243,70 @@ import alert from '../alert';
...
+        close = () => {
+            if (this.currentDialog) {
+                dialogHelper.close(this.currentDialog);
+            }
+        };
+    }
 
 /* eslint-enable indent */
-export default directoryBrowser;
+export {DirectoryBrowser};
```
Comment (thornbill) [inline src/components/directorybrowser/directorybrowser.js:312]: This seems like an unusual way of exporting this class.

File: .eslintrc.js (hunk 1 above)
Comment (grafixeyehero) [reply .eslintrc.js]: https://github.com/jellyfin/jellyfin-web/pull/2707/commits/d6c00aa6f40efbf41e0acd6c6398b6f356b787e5

File: src/components/directorybrowser/directorybrowser.js (hunk 2 above)
Comment (grafixeyehero) [reply src/components/directorybrowser/directorybrowser.js]: https://github.com/jellyfin/jellyfin-web/pull/2707/commits/d6c00aa6f40efbf41e0acd6c6398b6f356b787e5

**Review by thornbill — APPROVED** [review]

**Pull request merged**